src/routes/explore/explore.styles.ts

```diff
@@ -39,6 +38,7 @@ export const Viewer = styled.div<ViewerProps>`
   align-items: ${(props) => (props.fullRadar ? 'center' : 'flex-end')};
   justify-content: ${(props) => (props.fullRadar ? 'center' : 'flex-end')};
   width: ${(props) => (props.fullRadar ? '100%' : undefined)};
+  position: relative;
```

Comment:
Isn't this the line that causes zoom buttons jump to the side when zooming the radar? Without the `position` on the `Viewer`, everything works fine from what I've seen. 🤔